cypress/integration/attributes.js

```diff
@@ -0,0 +1,67 @@
+const AttributeList = ["Strength","Dexterity", "Stamina", "Charisma", "Manipulation", "Composure", "Intelligence", "Wits", "Resolve"]
+const lvl3Attrs = ["Dexterity", "Stamina", "Charisma"]
+const lvl2Attrs = ["Manipulation", "Composure", "Intelligence", "Wits"]    
+
+function checkNumberOfDots(Item, Expected) {
+    var i = 0;
+    var ExpectedResults = []
+    for (i; i < 5; i++) {
```

Comment:
nevidim ze by si pouzival premenu i mimo  tento cyklus
takze v tomto pripade asi moze byt for(var i=0; atd